### ISSUE TITLE
Compound Icons — Release Oct. 13

### ISSUE THREAD

**Issue by janogarcia** [vector-im/compound#245]
# Breaking changes

- `Check Circle` renamed to `Check Circle Solid`. `Check Circle` will still exist but now it will be pointing to the outline version, instead of the solid one.

# Platform

All.

```[tasklist]
# Release
- [x] Communicate breaking changes to release manager
- [ ] https://github.com/vector-im/compound-design-tokens/pull/60
- [ ] Communicate breaking changes to platform engineers
```

**Comment by germain-gg** [comment]
@janogarcia I can see you've assigned this to me? 
What actions do you want me to take? I'm also unsure why this references the 13th of October?

**Comment by janogarcia** [comment]
> What actions do you want me to take?

Just the second one, `Push new and updated icons from Compound Icons on Figma to Compound repository`.

>  I'm also unsure why this references the 13th of October?

We sort of agreed to sync the icons once per week, and I had to set a name for this release, but feel free to edit it.

> I don't think we need to "announce" changes to engineers. We are using semantic versioning and might want to have a system to add a changelog to the packages, but that's the extent of it.

Yeah, we are at a point where we could benefit from having more structured/visible changelogs. Unsure how would you prefer to set it up for packages, though.

So, currently, there's probably little visibility for changelogs, and even if we have proper ones, I think it's still a good idea to communicate breaking changes proactively in the internal channels. That's the goal for the last task, `Communicate breaking changes to platform engineers` (e.g., Sharing a quick release note on Compound Lobby).

**Comment by germain-gg** [comment]
Released! `@vector-im/compound-design-tokens@0.0.7`
